### DISCUSSION TITLE
map of country/continent based on event.location

### DISCUSSION THREAD

**Discussion by vdias38** [devizem/Eventure#93] [Ideas]
In a worldwide perspective seems interesting to show country and its position on continent

![Screenshot_20221005-130126_franceinfo](https://user-images.githubusercontent.com/9011612/194099358-781b934b-4724-46b2-8989-14f78f402df7.jpg)

**Comment by SirZemar** [comment]
This idea could already be implemented to instagram post. It could help fix the displacement of the country name text.
